<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGEditContentElementWidget</class>
 <widget class="QWidget" name="GGEditContentElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radText">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radImg">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAnim">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Animated Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radVideo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Video</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stkPages">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageText">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="txtText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageImage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
         <item>
          <widget class="QLineEdit" name="txtPathImage"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseImage">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>236</width>
            <height>253</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lblPreviewImage">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtPathImage</sender>
   <signal>editingFinished()</signal>
   <receiver>GGEditContentElementWidget</receiver>
   <slot>loadPreviewImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radText</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGEditContentElementWidget</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radImg</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGEditContentElementWidget</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radAnim</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGEditContentElementWidget</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radVideo</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGEditContentElementWidget</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadPreviewImage()</slot>
  <slot>changePage()</slot>
 </slots>
</ui>
